csrc/scheduler/pointwise_heuristic.h

```diff
@@ -43,6 +43,10 @@ class PointwiseParams : public HeuristicParams {
   // Unroll or vectorization factor
   size_t unroll_factor = 1;
 
+  // Reorder applied to reference tensor for scheduling, this is used to
+  // leverage allocation domain for efficient indexing / vectorization.
+  std::unordered_map<int, int> rfactor_reorder_map = {};
```

Comment:
This doesn't need to be in the heuristics. I was keeping it here to avoid re-computation. But I agree with you this isn't a good spot.

Wondering what would be a good place to store compile time info?

Reply:
I ended up just recomputing this...

Reply:
We do have something for caching compile time information https://github.com/NVIDIA/Fuser/blob/main/csrc/scheduler/compile_time_info.h
but unfortunately it is not always available. So recomputing it seems to be a valid approach...